lib/auto-languageclient.ts

```diff
@@ -39,21 +39,6 @@ import SignatureHelpAdapter from './adapters/signature-help-adapter';
 
 export type ConnectionType = 'stdio' | 'socket' | 'ipc';
 
-// Public: Defines the minimum surface area for an object that resembles a
```

Comment:
Decided to move this to ServerManager's file because ServerManager was still using ChildProcess to refer to the language server process.